### PR TITLE
[G2M] Buttons/fix

### PR DESCRIPTION
**Pull request by hyx131**
**Changes:**
- inherit `variant` & `size` prop from `ButtonGroup` wrapper
- add white bg to `outlined` buttons
- change existing `white` colour to `secondary-50`

### CODE REVIEW

File: stories/buttons/button-story.js [hunk 1]
```diff
@@ -74,68 +74,68 @@ const ButtonStory = ({ variant }) => {
...
             </ButtonGroup>
           </div>
           <div className='mt-5'>
-            <ButtonGroup variant={variant}>
-              <Button variant={variant} size='md'>Button</Button>
-              <Button variant={variant} size='md'>Button</Button>
-              <Button variant={variant} size='md'>Button</Button>
-              <Button variant={variant} size='md'><Add size='md' /></Button>
+            <ButtonGroup variant={variant} size='md'>
+              <Button size='md'>Button</Button>
```
Comment (geoerika) [inline stories/buttons/button-story.js:93]: We want to delete also the size prop in Button components in this group as well. Right?

**Review by geoerika — CHANGES_REQUESTED** [review]

File: stories/buttons/button-story.js (hunk 1 above)
Comment (hyx131) [reply stories/buttons/button-story.js]: whoops, too many missed that one lol, fixed!

**Review by sallkall — APPROVED** [review]

**Comment by geoerika** [comment]
Can we adjust the stories for buttons to actually show what code we use for buttons? Right now the Code section is not very helpful.
Screen Shot 2022-01-18 at 10 28 47 AM

**Comment by geoerika** [comment]
Yes, instead of using a button-story, you create a Template using the new syntax in Storybook: https://storybook.js.org/docs/react/writing-stories/introduction
The, use args to create the same sections as above.
For ex.
Template:
```const Template = (args) => ((
  <>
    <div className={styles.container}>
      <p className={styles.label}>Large:</p>
      <Button variant={args.variant} size='lg'>Button</Button>
      <Button variant={args.variant} size='lg' startIcon={<Add size='lg' />}>Button</Button>
      <Button variant={args.variant} size='lg' endIcon={<Add size='lg' />}>Button</Button>
      <Button variant={args.variant} size='lg'><Add size='lg' /></Button>
      <Button variant={args.variant} size='lg' type='success'>Success</Button>
      <Button variant={args.variant} size='lg' type='warning'>Warning</Button>
      <Button variant={args.variant} size='lg' type='error'>Error</Button>
      <Button disabled variant={args.variant} size='lg'>Disabled</Button>
    </div>....
 ```
And the story would be created by:
```
export const Outlined = Template.bind({})
Outlined.args = {
  variant: 'outlined',
}
```

**Comment by geoerika** [comment]
@Selin What do you think of ^?

**Comment by hyx131** [comment]
> @hyx131 What do you think of ^?

looks noice! applied, thanks 🙏

**Comment by geoerika** [comment]
> Storybook preview

The code looks fine, I just don't understand why the storybook misses the variant argument for outlined buttons. All the other variants have the argument added. Maybe a mistake of Chromatic??
Outlined:
Screen Shot 2022-01-18 at 12 43 54 PM
Elevated:
Screen Shot 2022-01-18 at 12 44 52 PM

**Review by geoerika — APPROVED** [review]

**Comment by hyx131** [comment]
> 

it's cuz `outlined` is the default variant lol

**Pull request merged**